src/components/footer.tsx

```diff
@@ -72,65 +72,45 @@ const AutoScrollLink: React.FC<any> = props => {
   return <Link onClick={scrollToContent} {...props} />
 }
 
-const classifiedAdminAreas = config.ADMIN_AREAS.map(area => ({
-  areaClass: `${regionId}-${area.toLowerCase()}`,
+const classifiedAdminAreas = config.ADMIN_AREAS.map(area => [
+  `${regionId}-${area.toLowerCase()}`,
   area
-}))
+])
 
 export const Footer: React.FC = props => {
   const { t, i18n } = useTranslation()
 
+  let aboutMenu: any = i18n.t('footer.aboutMenu', { returnObjects: true })
```

Comment:
Is there no way to provide a default / fallback here?

Reply:
Also, this could be typed as
`Array<[string, string]>`

Reply:
See https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple

Reply:
Yea, the `let` is a bit gross. I looked but didn't find a clean way to have a fallback unfortunately

Reply:
Not the end of the world, though the exists might make it easier to do a ternary. Regardless, type suggestion still applies :)